### ISSUE TITLE
Foundries dependecy

### ISSUE THREAD

**Issue by wero1414** [arduino/portenta-containers#2]
Hello, i tried to play with this cointainers but eventhough the tutorial (https://docs.arduino.cc/tutorials/portenta-x8/display-output-webgl) says this repository is an alternative from using foundriesFactories (which i dont want to pay).

<img width="661" alt="image" src="https://github.com/arduino/portenta-containers/assets/15166625/485d0fd8-5cba-480b-8dee-c04d7c887f9f">

This repository also depends on having a factory registered in their platform. 

<img width="757" alt="image" src="https://github.com/arduino/portenta-containers/assets/15166625/61efb1b2-1b5d-4495-a877-0229db09473b">
 
I tried putting the credentials to access the Factory "Arduino" but my device lacks of permits, so i dont know how to play with this containers.

<img width="1730" alt="image" src="https://github.com/arduino/portenta-containers/assets/15166625/4a12a4f3-2e0d-41b4-aa9c-fb0833e49d4c">

Do you have any plan on having the containers standalone, or in the docker hub?

Thanks!

**Comment by MaxPayne86** [comment]
@wero1414 you're right, I will check with the team about this portenta-containers repository. We build some of the images here and push them of dockerhub. You need to edit this [line](https://github.com/arduino/portenta-containers/blob/89ff896b07b94cb617d2bfd1e35495634e6b0f23/python-rpc-serial/docker-compose.yml#L3) to address [dockerhub images](https://hub.docker.com/r/arduino/python-rpc-serial/tags) and you're set. We need to duplicate the docker-compose.yml in something like docker-compose.foundries.io.yml and docker-compose.yml (which points to dockerhub Arduino).

**Comment by MaxPayne86** [comment]
> the subscription necessary to run a custom container is a $250/month

Foundries.io subscription is a fleet management service, you don't need such a service to run a custom container on the board. If it was otherwise advertised, then we have to fix the doc where we say that (in case plz provide a link)

 > because [the Arduino repository for this example is actually empty](https://hub.docker.com/r/arduino/python-ble-scanner-x8/tags)
 
 We have a [github workflow file](https://github.com/arduino/portenta-containers/blob/276af0649a5f39e163f8ad8410364383fd436abf/.github/workflows/docker-publish.yml#L38) please send me a PR to add this container there thanks. We will soon review this part on our end too, but it's probably the quickest way.
 
 Thanks for your feedback guys. Btw we're working on this url issue and we will update the discussion here asap

**Comment by wero1414** [comment]
I made my own docker images too and push them to dockerHub, as a solution, but maybe a lot of people will struggle with this

**Comment by wero1414** [comment]
> @wero1414 you're right, I will check with the team about this portenta-containers repository. We build some of the images here and push them of dockerhub. You need to edit this [line](https://github.com/arduino/portenta-containers/blob/89ff896b07b94cb617d2bfd1e35495634e6b0f23/python-rpc-serial/docker-compose.yml#L3) to address [dockerhub images](https://hub.docker.com/r/arduino/python-rpc-serial/tags) and you're set. We need to duplicate the docker-compose.yml in something like docker-compose.foundries.io.yml and docker-compose.yml (which points to dockerhub Arduino).

I checked Arduino Docker Hub and i see now you are pushing all the images right there, im happy to see this, hope the documentation soon gets an update, Thanks for the following

**Comment by MaxPayne86** [comment]
@rxbynerd @wero1414 we now build the docker images in CI and publish on Arduino dockerhub

https://github.com/arduino/portenta-containers/blob/c37d9ac11d058cfae4fb322d198f5a80364ec7cd/.github/workflows/docker-publish.yml#L26

we need to expand this list with what we currently support and then we have to switch the urls in the docker-compose files. Regarding the marketing stuff I will forward to the correct person, thanks for reporting.

> Portenta X8 has been bricked
Regarding this I woud suggest to open a support ticket if you didn't do it already, otherwise I would go with a reflash of the board.
